docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
```

Comment:
I tried and ended up getting stuck with a node build that didn't have the necessary node-gyp & python dev bits that were necessary to build our juttle node source. I can try again but it means having to fiddle with using the nodejs image + twiddling with the underlying system.